usr/lib/linuxmint/mintUpdate/mintUpdate.py

```diff
@@ -2544,14 +2544,16 @@ def open_preferences(self, widget, show_automation=False):
         page = SettingsPage()
         box.pack_start(page, True, True, 0)
         section = page.add_section(_("Package Updates"), _("Performed as root on a daily basis"))
-        autoupgrade_switch = Switch(_("Apply updates automatically"))
-        autoupgrade_switch.content_widget.set_active(os.path.isfile(AUTOMATIONS["upgrade"][2]))
-        autoupgrade_switch.content_widget.connect("notify::active", self.set_auto_upgrade)
-        section.add_row(autoupgrade_switch)
-        autodownload_switch = Switch(_("Download updates automatically"))
-        autodownload_switch.content_widget.set_active(os.path.isfile(AUTOMATIONS["download"][0]))
-        autodownload_switch.content_widget.connect("notify::active", self.set_auto_download)
-        section.add_row(autodownload_switch)
+        autoupgrade_combo = ComboBox(label = _("Select what to do automatically"), options = [[0, _("Nothing")], [1,_("Only Download")], [2,_("Download and install (recommended)")]])
```

Comment:
I'd not spell it like this though. I'd continue saying "Apply updates automatically" and name the options:
1. No (or Off)
2. Download only (Or only d/Download? Well, english is not my mother language, leaving that to you).
3. Either Yes or On. Well, Download and install could also be okay.
It's just that "Select what to do automatically" and "Nothing" maybe doesn't sound the best. @clefebvre

Reply:
Ah thanks, maybe my fault, think you changed it even before my comment - I were on the wrong commit, that's why my comment is marked as outdated! Thanks!